modbus: {baud: 19200, port: /dev/ttyUSB3, timeout: 10,
read_holding_registers_slaveids: [3],
read_holding_registers_count: [5],
read_holding_registers_start_address: [0],
read_input_registers_slaveids: [3],
read_input_registers_count: [8],
read_input_registers_start_address: [1000]}
